plugins/urls/__init__.py

```diff
@@ -338,356 +401,190 @@ def _respond_shorten(self, result, source, handler):
 
         if result is not None:
             return source.respond(result)
-        return source.respond(__("Unable to shorten using handler %s. Poke the"
-                                 "bot owner!")
-                              % handler)
+
+        source.respond("Unable to shorten using handler %s. Poke the bot "
+                       "owner!" % handler)
 
     def _respond_shorten_fail(self, failure, source, handler):
         """
         Respond to a shorten command, after a failed Deferred
         """
 
-        return source.respond(__("Error shortening url with handler %s: %s")
-                              % (handler, failure))
+        source.respond("Error shortening url with handler %s: %s" % (
```

Comment:
Stop printing exceptions to IRC. A) Exceptions are internal things, and random people on the internet have no need to know them, B) random people on the internet can do nothing with that information except tell the bot owner, who you can give information to with a log call.